<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QMainWindow" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Participante</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Peso</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Kg</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="peso">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>150</number>
      </property>
     </widget>
     <widget class="QFrame" name="bandera">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(C:/Users/USUARIO/Desktop/Servicio_Comunitario/images/banderas/alemania.png) 0 0 0 0 stretch stretch;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="fecha_nacimiento">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pais / Nacionalidad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202, 38, 62);
border: 2px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/recursos/papelera.png</normaloff>images/recursos/papelera.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QFrame" name="foto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Altura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="altura">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="cargar_foto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar Foto</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edad">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>170</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha de Nacimeinto</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="apellido">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="guardar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 214, 25);
border: 2px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/recursos/listo.png</normaloff>images/recursos/listo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pais">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="asignar_rojo">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
border: 2px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/recursos/person.png</normaloff>images/recursos/person.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="asignar_azul">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 255, 220);
border: 2px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/recursos/person.png</normaloff>images/recursos/person.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>300</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="estado">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="estado_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="estado_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>Asignar a Rojo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="estado_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Asignar a Azul</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="vaciar_campos">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 0);
border: 2px solid black;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/recursos/borrador.png</normaloff>images/recursos/borrador.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="estado_5">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>190</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vaciar Campos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dfs">
     <attribute name="title">
      <string>Cargar Datos</string>
     </attribute>
     <widget class="QPushButton" name="cargar_datos">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar Datos</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabla_participantes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>430</width>
        <height>200</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nombre Completo</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Edad</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peso (Kg)</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Altura (Cm)</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">id</string>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="buscador">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Configuracion del Sistema</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <action name="actionsdfs">
   <property name="text">
    <string>sdfs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
